sys/net/application_layer/cord/epsim/cord_epsim.c

```diff
@@ -60,9 +66,14 @@ int cord_epsim_register(void)
     }
     /* finish, we don't have any payload */
     ssize_t len = gcoap_finish(&pkt, 0, COAP_FORMAT_NONE);
-    if (gcoap_req_send2(buf, len, &remote, NULL) == 0) {
+    if (gcoap_req_send2(buf, len, rd_ep, _req_handler) == 0) {
         return CORD_EPSIM_ERROR;
     }
 
     return CORD_EPSIM_OK;
```

Comment:
I would expect to set/return CORD_EPSIM_BUSY here.

Reply:
Indeed I missed to set the `_state` to busy, that is fixed now. 

But for the return value, I still think returning `_OK` here makes sense, to signal that the registration request was sent our correctly.

Reply:
Yes, I agree. The return value needs to distinguish when the module truly was busy, as at the top of the register function.